runtime/include/chpl-tasks.h

```diff
@@ -228,6 +228,20 @@ c_sublocid_t chpl_task_getRequestedSubloc(void);
 chpl_taskID_t chpl_task_getId(void);
 
 //
+// Checks whether two task IDs are the same
+//
+chpl_bool chpl_task_idEquals(chpl_taskID_t, chpl_taskID_t);
+
+//
+// Returns the string representation of task ID
+// The string returned is the same buffer passed as argument
```

Comment:
I take that back. I think it should return whether or not the operation was successful. E.g. return 0 if the task ID was encoded in the string, or 1 if there was not enough space.